cmds/core/yes/yes.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+)
+
+func main() {
+	args := os.Args
+	yOutput := "y"
+	if len(args) == 0 {
```

Comment:
Since you have a default string to print when the user doesn't give any arguments ("y"), it wouldn't be an error if `len(flag.Args())` was 0; instead maybe what you could do is check if `len(flag.Args()) > 0` and if it is, replace `yOutput` with whatever the user provided, and if not, continue with the default value of `yOutput`

Reply:
Sounds good, I just fixed it. I realized using flag.Args definitely resolved many of my checks.